bin/physlr-make

```diff
@@ -41,14 +41,18 @@ endif
 endif
 
 # Minimum number of overlapping minimizers
-ifndef minimizer_overlap
-m=90
-else ifeq ($(minimizer_overlap), stLFR)
+ifdef protocol
+ifeq ($(protocol), stlfr)
 m=85
-else ifeq ($(minimizer_overlap), 10X)
+else ifeq ($(protocol), 10x)
 m=92.5
+endif
 else
+ifdef minimizer_overlap
 m=$(minimizer_overlap)
+else
+m=90
```

Comment:
Can you remind me why we need both `m` and `minimizer_overlap`? It seems to me just having `protocol` and `m` would suffice?

Reply:
the idea was the user would be more informed if he uses the long form of `m` which is `minimizer_overlap` but internally we use` m`, I could change all the `m` to `minimizer_overlap` or vice verse

Reply:
Right.. I suppose the name isn't as descriptive now that we are removing a % of edges vs. using an absolute minimizer threshold. I'd be OK with changing it all to `m` -- the help page is there for the user to understand what it controls.